js/samples/04.ai-apps/a.teamsChefBot/src/prompts/default/config.json

```diff
@@ -16,7 +16,7 @@
         "stop_sequences": []
     },
     "augmentation": {
-        "augmentation_type": "none",
+        "augmentation_type": "default",
```

Comment:
aren't default and none the same?

Reply:
yup, just wanted to make it more explicit since streaming only works with default